README.md

```diff
@@ -1,16 +1,38 @@
-# EVO-Exchange
+# Obmenyashka (Child Goods Exchange)
 
-This is the back-end part of the EVO Hillel project "Obmenyashka". 
+This is the back-end part of the IT-Hillel EVO project "Obmenyashka".
+
+We want to give our users opportunity to share or exchange any children clothes.  
 
 The technologies used:
-- Java, Spring Boot, Spring Web MVC, Spring Data JPA, Spring Security, JWT; 
-- MySQL, Liquibase.
+- Java, Spring (Boot, Web MVC, Data JPA, Security), JWT, Lombok 
+- MySQL, Liquibase, Database-Rider
+- Swagger
+- Docker
 
 ## Usage
 ### Requirements
 
- - [JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html) `version 8+`
- - Local MySQL server with scheme named `evo_exchange`
+ - [JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html) `version 11`
+ - [MySQL](https://www.mysql.com/downloads/) `version 8`
+ - [maven](https://maven.apache.org/index.html) `version 3.6.+`
+ - MySQL scheme `evo_exchange` will be created after the very first local run
+
+### Installation
+
+Use a project builder [maven](https://maven.apache.org/index.html) to install the project if you fulfil the requirements.
+```bash
+mvn install
```

Comment:
It may be better to use the 'clean install' command to clean up the project's working directory of old files.

Reply:
Ok